### DISCUSSION TITLE
Impact of Force-Pushing on Existing Pull Requests

### DISCUSSION THREAD

**Discussion by Frizk1429** [community/community#142466] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

What happens if you create a pull request from a branch that was force-pushed after the PR was opened?

**Comment by fxrdhan** [comment, marked as answer]
When you force-push to a branch with an open pull request, the PR automatically updates to reflect the new commit history. However, this can cause confusion for reviewers since their previous comments might no longer match the current code context. GitHub will mark previous reviews as outdated. It's generally considered a bad practice to force-push to branches with open PRs unless absolutely necessary.
